packages/twenty-server/src/workspace/messaging/repositories/message/message.service.ts

```diff
@@ -203,12 +203,30 @@ export class MessageService {
         workspaceId,
       );
 
-    if (isContactAutoCreationEnabled) {
+    const isOutgoing = connectedAccount.handle === message.fromHandle;
```

Comment:
Makes me think we should draw the line at some point and clearly define when to use connected account vs channel. In this case both store the handle for example and I think "messaging" should be closer to channel than connected account.